d7a/system_files/system_files.py

```diff
@@ -33,6 +33,8 @@
 
 
 class SystemFiles:
+  enum_class = SystemFileIds
+
   files = {
     SystemFileIds.UID: UidFile(),
```

Comment:
The classes don't need to be instantiated here, as their main use will be to use the static method try_parse. In this case the class name is enough

Reply:
Not sure what you mean, these files are also used in the parse_hexstring where it does get used as a class. This is also not code that I changed.

Reply:
nvm. The length field is actually accessed in some places, which is only defined on instance level